sechub-pds-examples/gosec/container-create.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+if [[ -z "$1" ]]
+then
+  echo "Please provide a docker registry server."
+  exit 1
+fi
+
+if [[ -z "$2" ]]
+then
+  echo "Please provide a version for the container."
+  exit 1
+fi
+
+REGISTRY="$1"
+VERSION="$2"
+BASE_IMAGE="$3"
+
+if [[ -z "$BASE_IMAGE" ]]
+then
+    BASE_IMAGE="ubuntu:focal"
```

Comment:
Is there a special reason to take the Ubuntu image?
You could take e.g. an Alpine image and save ~200M of image size.

In the end, we want to provide a Java environment with AdoptOpenJDK.

Reply:
There are several reasons to take the Ubuntu image:

* Ubuntu Focal is a LTS release version (5 year public support)
* Most software which works under Linux was either tested with or developed for Ubuntu
* Most people have worked with Debian or Ubuntu in some form -> low barrier of entry
* There are supporting resources in form of blogs, forums etc. about Ubuntu everywhere
* Ubuntu uses libc instead of musl as standard library -> avoid any issues which may occur because of it

As you pointed out, using Alpine Linux would lower the image size, which is in my opinion an optimization.

Reply:
We will provide a Dockerfile and scripts to build an Alpine image for advanced users which want to run PDS+GoSec in Kubernetes.